src/Stringy.php

```diff
@@ -427,7 +427,7 @@ public function toAscii()
     {
         $str = $this->str;
         $charsArray = array(
-            'a'    => array('à', 'á', 'â', 'ã', 'ā', 'ą', 'ă', 'å', 'α', 'ά', 'ἀ',
+            'a'    => array('à', 'á', 'â', 'ä', 'ã', 'ā', 'ą', 'ă', 'å', 'α', 'ά', 'ἀ',
```

Comment:
I know, they already were, and the orderings are a bit inconsistent too. I just didn't touch this to keep the changes the simplest possible.